### PR TITLE
Vignette documentation draft and README updates

### PR DESCRIPTION
**Pull request by brendoncampbell**

### CODE REVIEW

File: vignettes/Rmagine-vignette.Rmd [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+Here's an example photo we'll be using taken at Coronado Beach, San Diego:
+![Sample Image](imgs/coronado_beach.jpeg)
+
+### Using the Tunnel Filter
+
+This function produces an image with strong visual distortion intended to create a tunnel or pincushion effect.
+
+```r
+library(Rmagine)
+tunnel_filter(image_path, k, rot)
```
Comment (katieb1) [inline vignettes/Rmagine-vignette.Rmd]: Removed distortion coefficient
```suggestion
tunnel_filter(image_path, rot)
```

File: vignettes/Rmagine-vignette.Rmd [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+```
+
+#### Arguments
+
+- `image_path`: file path of the input image (string)
+- `k`: distortion coefficient (float, optional). If not specified, defaults to 0.5
```
Comment (katieb1) [inline vignettes/Rmagine-vignette.Rmd]: Removed distortion coefficient
```suggestion
```

File: vignettes/Rmagine-vignette.Rmd [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+- `rot`: rotation degree holding value between -0.5 and 0.5 (float, optional). If not specified, defaults to 0.5
+
+#### Output
+
+- Saves the image with tunnel filter applied to the current working directory
+
+#### Example
+
+- `tunnel_filter("imgs/coronado_beach.jpeg", 0.5, -0.2)`
```
Comment (katieb1) [inline vignettes/Rmagine-vignette.Rmd]: Removed distortion coefficient
```suggestion
- `tunnel_filter("imgs/coronado_beach.jpeg", -0.2)`
```

**Review by katieb1 — CHANGES_REQUESTED** [review]

**Review by Sukriti1312 — APPROVED** [review]
Looks good.

**Pull request merged**